deploy/batch_account.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python3
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import argparse
+import json
+import subprocess
+
+parser = argparse.ArgumentParser(description='Arguments required to run granting access to batch account')
```

Comment:
Should we simplify this script in terms of the number of parameters required to run the script? Synapse workspace name, synapse workspace resource group, batch account uami, batch account resource group, role can be derived if the use passes the environment Code. Similar to the implementation in configure.sh. The goal is to simplify the use of the scripts.

Reply:
Fixed